configure.ac

```diff
@@ -292,11 +292,11 @@ DOWNLOAD=":"
 DOWNLOAD_TO_STDOUT=
 DOWNLOAD_TIMEOUT=
 if test "x${WGET}" != "x:"; then
-  DOWNLOAD=${WGET}
+  DOWNLOAD="${WGET} -L"
```

Comment:
wget already follows redirects, the -L option just does:

  -L,  --relative                  follow relative links only

So this is for the case where curl is the only thing available.